src/Microsoft.AspNetCore.Sockets/ConnectionManager.cs

```diff
@@ -66,16 +66,19 @@ public DefaultConnectionContext CreateConnection()
             var applicationSide = ChannelConnection.Create<byte[]>(transportToApplication, applicationToTransport);
 
             var connection = new DefaultConnectionContext(id, applicationSide, transportSide);
+            var connectionTimer = SocketEventSource.Log.ConnectionStart(id);
```

Comment:
Where is the ILogger.Log for this?

Reply:
On line 60, I'll move this to be next to it.